industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,8 @@
+1.  Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were converted to NME.
```

Comment:
Protonation states haven't changed, but all the protons were replaced it seems - could you confirm that this is the default behaviour with the instructions' prepwizard pre-process step?

Reply:
By comparison the protons weren't re-assigned in jnk1.

Reply:
here you mean the Atom type names, right? :)

Reply:
No the protons were fully reassigned - i.e. all their coordinates subtly changed.